com.github.danrabbit.harvey/com.github.danrabbit.harvey.yml

```diff
@@ -0,0 +1,30 @@
+id: com.github.danrabbit.harvey
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: com.github.danrabbit.harvey
+rename-icon: com.github.danrabbit.harvey
+tags: ['contrast', 'color']
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--talk-name=ca.desrt.dconf'
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  - '--filesystem=xdg-run/dconf'
+  - '--filesystem=~/.config/dconf:ro'
+  - '--own-name=com.github.danrabbit.harvey'
```

Comment:
Again, the ID seems to be correct, this shouldn’t be needed. Try removing it and seeing if everything still runs OK.